WowCharComparerWebApp/Controllers/UserControllers/RegisterController.cs

```diff
@@ -30,21 +33,19 @@ public IActionResult Index()
         [HttpGet]
         public IActionResult Confirmation(string userID)
         {
+            try
             {
-                try
-                {
-                    DbOperationStatus<User> operationStatus = _dbAccessUser.ActivateAccount(userID);
+                DbOperationStatus<User> operationStatus = _dbAccessUser.ActivateAccount(userID);
 
-                    if (!operationStatus.OperationSuccess)
-                        throw new Exception("Error while activating account");
-                }
+                if (!operationStatus.OperationSuccess)
+                    throw new Exception("Error while activating account");
+            }
 
-                catch (Exception ex)
-                {
-                    return View("Error", ex);
-                }
-                return View("AccountApproval");
+            catch (Exception ex)
+            {
+                return View("Error", ex);
             }
+            return View("AccountApproval");
```

Comment:
Would be great to return here GenericUserView, same as in LoginController

Reply:
Sure forget about this method!

Reply:
In the LoginController it may be some problem to fit generic view for all of method. PasswordRecovery and ChangeUserPassoword are specific, because they redirecting to views where whole login/password/email input is required.

Reply:
If it gonna be hard, we can forsake by now, remember to add it as task in trello. We should take care of it later.